.github/workflows/ci-docker.yml

```
@@ -0,0 +1,175 @@
+        id: load
+        uses: ./.github/actions/load-docker-image
+        with:
+          is-fork: ${{ needs.build.outputs.is-fork }}
+          image-tags: ${{ needs.build.outputs.image-tags }}
+
+      - name: Inspect image size and layers
+        shell: bash
+        run: |
+          IMAGE_TAG="${{ steps.load.outputs.image-tag }}"
+          echo "Analyzing Docker image: $IMAGE_TAG"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)